usage.c

```diff
@@ -15,7 +15,12 @@ void vreportf(const char *prefix, const char *err, va_list params)
 
 	fflush(fh);
 	if (!tweaked_error_buffering) {
+#if defined(_MSC_VER)
+		/* UCRT doesn't like zero buffer size */
+		setvbuf(fh, NULL, _IOLBF, BUFSIZ);
+#else
 		setvbuf(fh, NULL, _IOLBF, 0);
+#endif
```

Comment:
@KindDragon But that would revert https://github.com/jeffhostetler/git/commit/f4c3edc0b156362a92bf9de4f0ec794e90a757fc. In other words, you suggested a regression for platforms other than Windows, at least.